lib/Constants.js

```diff
@@ -370,5 +370,244 @@ module.exports = {
   },
   commandClass (cmd) {
     return this._commandClassMap[cmd] || 'unknownClass_' + cmd
+  },
+  _genericDeviceClassMap: {
+    // https://github.com/home-assistant/core/blob/dev/homeassistant/components/zwave/const.py#L196
+    // 0x00: specific_type_not_used // Available in all Generic types
+    0x03: {
+      genericType: 'generic_type_av_control_point',
```

Comment:
I think you could use just `generic` and `specific` as keys

Reply:
Fixed in ff15b8dc922b163dc5130109d38ad679ba1f4439